integration_test/querymonkey/test_queries/neighbor_count.sql

```diff
@@ -1,4 +1,4 @@
-SELECT 'monkey', o1.id AS id1, o2.id AS id2,
+SELECT 'monkey', count(o1.id1),
   DISTANCE(POINT('ICRS', o1.coord_ra, o1.coord_decl), POINT('ICRS', o2.coord_ra, o2.coord_decl)) AS d
```

Comment:
I don't think you want both `COUNT()` (aggregate function) and `DISTANCE()` (scalar function) in the same `SELECT` clause.

Reply:
Okay sounds good thanks!